cmd/artifact/install/install.go

```diff
@@ -97,6 +97,10 @@ func (o *artifactInstallOptions) RunArtifactInstall(ctx context.Context, args []
 			o.Printer.CheckErr(fmt.Errorf("unable to retrieved the configured installer: %w", err))
 		}
 
+		// Set args as configured if no arg was passed
+		if len(configuredInstaller.Artifacts) == 0 {
+			return fmt.Errorf("no artifacts to install, please configure artifacts or pass them as arguments to this command")
+		}
```

Comment:
This is ok for the args and refs of the config. However, there is another point to address: here we are considering the config only if `args` len is equal to zero. But this is not completely correct, since we want to load the config in any case because it may contain other parameters, like for instance the plugins dir and rulesfiles dir. So I would write something like this at the beginning of installer:
```
	// Retrieve configuration for installer
	configuredInstaller, err := config.Installer()
	if err != nil {
		o.Printer.CheckErr(fmt.Errorf("unable to retrieved the configured installer: %w", err))
	}

	// Set args as configured if no arg was passed
	if len(args) == 0 {
		if len(configuredInstaller.Artifacts) == 0 {
			return fmt.Errorf("no artifacts to install, please configure artifacts or pass them as arguments to this command")
		}
		args = configuredInstaller.Artifacts
	}
```

Reply:
Aah! That's right! Thank you! :D